Makefile

```diff
@@ -4,6 +4,7 @@
 VERSION = $(shell git tag -l|tail -n 1|sed -e 's/^v//')
 DESTDIR = /usr
 SCRIPTS = scripts/nvim.vmux scripts/nvim-qt.vmux scripts/vim.vmux scripts/gvim.vmux scripts/kak.vmux
+MAIN = vmux/__main__.py
```

Comment:
Please file the changes in this file as separate pull request.
In total, please create three pull requests:
- pip inthegration
- neovim → pynvim
- compatibility with non-gnu cp

Reply:
will do - I'll send something through later today

Reply:
done :)